packages/@apphosting/adapter-nextjs/src/interfaces.ts

```diff
@@ -102,10 +101,52 @@ export interface OutputBundleOptions {
   outputStaticDirectoryPath: string;
 }
 
-// Metadata schema for bundle.yaml outputted by next.js adapter
-export interface Metadata {
+// Metadata schema for adapter metadata
+export interface AdapterMetadata {
   adapterPackageName: string;
   adapterVersion: string;
+}
+
+// Metadata schema for bundle.yaml outputted by next.js adapter
+export interface Metadata extends AdapterMetadata {
   framework: string;
   frameworkVersion: string;
 }
+
+// Metadata schema for asset bindings
```

Comment:
I don't think Metadata schema fits for all of these interfaces? I think these are all interfaces from the middleware-manifest.json file so it would be nice if that was clearer in the comments